frontends/concrete-python/version.txt

```diff
@@ -1 +1 @@
-__version__ = "0.0.0"
+__version__ = "1.0.0"
```

Comment:
??

Reply:
good catch, that should not have impact.